javascript/eslintrc.js

```diff
@@ -14,6 +14,7 @@ module.exports = {
     "no-shadow-restricted-names": 2,
     "no-undef": 2,
     "no-use-before-define": 2,
+    "quotes": [2, "double", "avoid-escape"],
```

Comment:
We're using ES6 on `browser`: I'm trying to check that this rule will work as expected in that repo. I'm guessing/hoping `"env": "es6"` was not on wherever you tested that?

My expectation/hope for an ES6 repo is that <pre>`static text`</pre> would be a violation of this rule, but <pre>`text with ${substitution}`</pre> would not be.